libcasr/src/csharp.rs

```diff
@@ -0,0 +1,395 @@
+//! C# module implements `ParseStacktrace` and `Exception` traits for C# reports.
+use crate::error::*;
+use crate::exception::Exception;
+use crate::execution_class::ExecutionClass;
+use crate::stacktrace::{ParseStacktrace, Stacktrace, StacktraceEntry};
+use regex::Regex;
+use std::vec::IntoIter;
+
+/// Structure provides an interface for processing the stack trace.
+pub struct CSharpStacktrace;
+
+struct FormatRegexIterator<'a> {
```

Comment:
Is it really necessary to use such strange structures?

Reply:
Fixed